hi-ml-histopathology/testhisto/testhisto/mocks/base_data_generator.py

```diff
@@ -104,15 +111,38 @@ def get_dataloader(self) -> Optional[DataLoader]:
         else:
             raise NotImplementedError
 
+    def generate_mock_histo_data(self) -> None:
+        """Create mock histo data and save it in the corresponding format: tiff for wsi and png for tiles"""
+        raise NotImplementedError
+
+    def mount_pathmnist_dataset(self) -> None:
+        ws = get_workspace()
+        dataset = DatasetConfig(
+            name=self.mock_type.value, target_folder=self.tmp_path / self.mock_type.value, use_mounting=True
+        )
+        dataset_mount_folder, mount_ctx = dataset.to_input_dataset_local(ws)
+        assert mount_ctx is not None  # for mypy
+        mount_ctx.start()
+        print(f"Dataset mounted in {dataset_mount_folder}")  # TODO remove this print
```

Comment:
this is a good print statement, I'd leave it there.

Reply:
Agree, I'm using logging.info to avoid having a print(...), I haven't pushed it yet